pkg/planner/core/indexmerge_path.go

```diff
@@ -108,6 +108,8 @@ func (ds *DataSource) generateIndexMergePath() error {
 		}
 	}
 
+	ds.cleanAccessPathForMVIndexHint()
```

Comment:
Looks like this function will interfere with the logic below and we may need to move it to the end of this function.

Reply:
Done.